packages/guides-restructured-text/tests/unit/Parser/Productions/ListRuleTest.php

```diff
@@ -26,7 +25,7 @@ final class ListRuleTest extends RuleTestCase
     protected function setUp(): void
     {
         $ruleContainer = $this->givenCollectAllRuleContainer();
-        $this->rule = new ListRule($ruleContainer, new NullLogger());
+        $this->rule = new ListRule($ruleContainer);
```

Comment:
please leave the tests untouched, whenever you need to change a unittest that might indicate a breaking change :-)

Reply:
done